zathura/zathura.c

```diff
@@ -546,6 +546,49 @@ document_info_open(gpointer data)
   return FALSE;
 }
 
+const char*
+get_formatted_filename(zathura_t* zathura, const char* file_path, bool statusbar)
+{
+  bool basename_only = false;
+  if (statusbar) {
+    girara_setting_get(zathura->ui.session, "window-title-basename", &basename_only);
+  } else {
+    girara_setting_get(zathura->ui.session, "statusbar-basename", &basename_only);
+  }
+
+  if (basename_only == false) {
+
+    bool home_tilde = false;
+    if (statusbar) {
+      girara_setting_get(zathura->ui.session, "statusbar-home-tilde", &home_tilde);
+    } else {
+      girara_setting_get(zathura->ui.session, "window-title-home-tilde", &home_tilde);
+    }
+
+    if (home_tilde) {
+      char *home = getenv("HOME");
```

Comment:
According to girara documentation (https://pwmt.org/projects/girara/doxygen/utils_8c.html#a71314474e74a8da69cafaab9155d0f12), $HOME is preferred to girara_get_home_directory. So maybe we should continue using $HOME.

Anyway, if you think it's better to use `girara_get_home_directory`, I can do that. I have a question though: girara_get_home_directory takes a `const char* user` as a parameter. Where can I get this info? It looks like girara_fix_path tries to find the pattern `~/(.*)/` in the path to find the username. This seems kind of sloppy. Is there a better way?